proto/proto/store/rpc.proto

```diff
@@ -259,6 +262,50 @@ message SyncStateResponse {
     repeated note.NoteSyncRecord notes = 7;
 }
 
+// SYNC ACCOUNT VAULT
+// ================================================================================================
+
+// Account vault synchronization request.
+//
+// Allows clients to sync asset values for specific public accounts within a block range.
+message SyncAccountVaultRequest {
+    // Block number from which to start synchronizing.
+    fixed32 block_from = 1;
+    
+    /// Block number up to which to sync. If not specified, syncs up to the latest block.
+    ///
+    /// If specified, this block must be close to the chain tip (i.e., within 30 blocks),
+    /// otherwise an error will be returned.
+    optional fixed32 block_to = 2;
+    
+    // Account for which we want to sync storage maps.
+    account.AccountId account_id = 3;
+}
+
+message SyncAccountVaultResponse {
+    // The block number of the last row included in this response.
+    // For chunked responses, this may be less than request.block_to.
+    fixed32 block_num = 1;
```

Comment:
nit: the reference to "row" here may be confusing. Could we say "last update" or something like that instead?

Should we also mention that if this is not equal to `request.block_to` then the user would need to make another request with `request.block_from` equal to `block_num + 1`?